index.html

```diff
@@ -6,6 +6,6 @@
   <meta nane="viewport" content="width=device-width, initial-scale=1.0">
 </head>
 <body>
-  
```

Comment:
Could you remove the 'Enjoy your stay.' string and make the 'welcome to Potionfy!' string red?

Reply:
sure can